genomedata/_load_seq.py

```diff
@@ -214,6 +220,80 @@ def get_num_seq(filenames):
 
     return res
 
+
+def load_chromosome_name_mapping(assembly_report_file):
+    res = {}
+
+    # Assume commented line above the first non-commented line is the header
+    # containing the field names
+    header_line = ""
+
+    # NB: These csv files are assumed to be small enough to be held in memory
+    assembly_report_lines = assembly_report_file.readlines()
+
+    # For every line in the file
+    header_index = 0
+    for line_index, line in enumerate(assembly_report_lines):
+        # If the header line has not been defined
+        if not header_line:
+            # Assume it is the first line
+            header_line = line
```

Comment:
Since you are assuming you've already read all the lines into memory anyway you can save some complexity by just having this loop get the index rather than updating header_line a bunch of times in the loop